src/components/EventsList/EventsList.js

```diff
@@ -12,19 +12,34 @@ function renderEvents(events) {
     />
   );
 
+  let noEventText = 'No events are coming up';
```

Comment:
It seems like this logic should live inside the `if` statement below on line 25.  

A small thing but there should be spaces after each of the `...`s and before the `Try`s in the text.